library/data/recipe/src/main/java/org/quiltmc/qsl/recipe/api/RecipeManagerHelper.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.recipe.api;
+
+import net.minecraft.recipe.Recipe;
+
+import org.quiltmc.qsl.recipe.impl.RecipeManagerImpl;
+
+/**
+ * Represents a helper for the {@link net.minecraft.recipe.RecipeManager}.
+ *
+ * @version 1.0.0
+ * @since 1.0.0
+ */
+public final class RecipeManagerHelper {
+	private RecipeManagerHelper() {
+		throw new UnsupportedOperationException("RecipeManagerHelper only contains static definitions.");
+	}
+
+	/**
+	 * Registers a static recipe.
+	 * <p>
+	 * A static recipe is a recipe that is registered at mod startup (or later) and is kept during the whole lifecycle
```

Comment:
Clarify when recipes can no longer be registered

Reply:
Er, there's no actual recipe registration, it's never locked at one point.

I probably should say that any recipe registered while the game is running  will only apply for the next data pack reload.